Update sizes-hpc.md

shirgall opened the pull request:
Update to indicate only Intel MPI 5.x versions are supported. (jopoulso@microsoft.com)

azurecla commented:
Hi __@tiagocostapt__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@tamram - Tamra, could you review Tiago's update?

tamram commented:
Looks good, thanks.

#sign-off

shirgall committed:
Update sizes-hpc.md

Update to indicate only Intel MPI 5.x versions are supported. (jopoulso@microsoft.com)
